src/QirRuntime/lib/QIR/intrinsicsOut.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <iostream>
+
+#include "qirTypes.hpp"
+#include "quantum__qis.hpp"
+
+// Forward declarations:
+static std::ostream& GetQOstream();
+
+// Public API:
+extern "C"
+{
+    void quantum__qis__message__body(QirString* qstr)   // NOLINT
+    {
+        GetQOstream() << qstr->str << std::endl;
+    }
+}   // extern "C"
+
+
+// Internal API:
+static  std::ostream * currOStream = &std::cout;    // Log to std::cout by default.
```

Comment:
Is the convention to write pointers' declarations like `static std::ostream * currOStream = &std::cout;` instead of `static std::ostream* currOStream = &std::cout;` (the `*` next to the type with no spacing).

Reply:
As far as I know (AFAIK) there is no such a convention in our C++ code. What I know is that there are at least 2 significant groups of C/C++ devs with different preferences.

One group prefers writing `int* i;`, reasoning it with separation of the type and variable name.

Another group prefers writing `int *i;` reasoning it that `*` changes the variable rather than a type, and if we write `int* i, j;` then only the first variable `i` will be the pointer, but the second variable `j` will not be a pointer (it will be `int`). And such a fragment can mislead those who are new to C/C++. But if we write `int *i, j;` then it will mislead less (by being closer to the reality).

Knowing that I personally prefer writing something _intermediate_ ;-) Both for pointers `*` and references `&`.

Reply:
Please run clang formatter -- it will do automated formatting that at some point we've agreed on for this project...

Reply:
Fixed.